views/tables/Table/index.jsx

```diff
@@ -8,6 +8,7 @@ import {
 } from '@mui/material'
 
 import TableHead from './TableHead'
+import { max } from 'lodash'
```

Comment:
do we need this import?

Reply:
No must be an accidentally IDE import. Will fix!

Reply:
2052a40